src/test/java/org/fundacionjala/coding/jimmy/Kata8Test.java

```diff
@@ -0,0 +1,92 @@
+package org.fundacionjala.coding.jimmy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Class to test the numbers and digits of OCR Bank.
+ */
+public class Kata8Test {
+
+    public static final String BLANK_LINE = "                           ";
+    private Map<String, String> useCase = new HashMap<>();
+
+    /**
+     * Method to assign the data for perform the test.
+     */
+    @Before
+    public void assignDataForTesting() {
+
+        useCase.put("000000000",
+                " _  _  _  _  _  _  _  _  _ "
+                        + "| || || || || || || || || |"
+                        + "|_||_||_||_||_||_||_||_||_|"
+                        + BLANK_LINE);
+
+        useCase.put("111111111",
+                BLANK_LINE
+                        + "  |  |  |  |  |  |  |  |  |"
+                        + "  |  |  |  |  |  |  |  |  |"
+                        + BLANK_LINE);
+
+        useCase.put("123456789",
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "123456789" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)